### PR TITLE
[ADZ-1180] Rebranding - document title banner

### PR DESCRIPTION
**Pull request by urchy**
Document title banner added to api-catalogue template.

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl [hunk 1]
```diff
@@ -17,24 +18,16 @@
 <#-- Content Page Pixel -->
 <@contentPixel document.getCanonicalUUID() document.title></@contentPixel>
 
+<@headerBanner document />
```
Comment (harryhmc) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl:21]: comparing this to `general-article.ftl` suggests that the banner image should be _inside_ the article, just above the grid-wrapper div

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl [hunk 2]
```diff
@@ -17,24 +18,16 @@
...
-        </div>
-        APi Catalogue - Preview
         <div class="grid-row">
             <div class="column column--two-thirds column--reset">
                 <#-- [FTL-BEGIN] 'Summary and optional Body' sections -->
                 <div id="section-summary" class="article-section article-section--summary  no-border">
                     <div class="grid-row">
                         <div class="column column--reset">
                             <div class="rich-text-content">
-                                <div data-uipath="website.linkslist.summary"><@hst.html hippohtml=document.summary contentRewriter=gaContentRewriter/></div>
```
Comment (harryhmc) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl:37]: dont think i understand why this has been removed?

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl (hunk 1 above)
Comment (urchy) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl]: You right, I was taking a look at the list-article which has it outside of the article.
I have pushed another version with it inside the article.

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl (hunk 2 above)
Comment (urchy) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl:37]: This is now part of the headerBanner. its the same info

File: repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl (hunk 1 above)
Comment (ziemsky) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/api-catalogue-preview.ftl]: Good spot, Harry.

Having said that: I know I kept mentioning `general-article.ftl` as the main reference, but having looked more closely into this with Andre, it seems that `nhsd-list-article.ftl` is probably a better basis for API catalogue page.

In `nhsd-list-article.ftl` the `<@headerBanner document />` is not wrapped in any container element whilst in `general-article.ftl`. Based on this, if the page looks okay in testing with the header wrapped in another container element I think it ultimately doesn't matter much and we can go ahead with either variant.

**Review by ziemsky — APPROVED** [review]
:+1:

**Pull request merged**